srcpkgs/python3-poetry-core/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-poetry-core'
+pkgname=python3-poetry-core
+version=1.0.3
+revision=1
+wrksrc="poetry-core-${version}"
+build_style="python3-pep517"
+make_install_target="poetry_core-${version}-*-*-*.whl"
+hostmakedepends="python3-wheel"
+depends="python3"
+short_desc="Poetry PEP 517 Build Backend & Core Utilities"
+maintainer="Kye Shi <shi.kye@gmail.com>"
+license="MIT"
+homepage="https://github.com/python-poetry/poetry-core"
+distfiles="https://github.com/python-poetry/poetry-core/archive/refs/tags/${version}.tar.gz"
```

Comment:
For Python distfiles, we generally prefer `${PYPI_SITE}` to source repos.

Reply:
Fixed in bbdc83d.